lib/routes/routePUT.js

```diff
@@ -19,7 +20,31 @@ export default function routePUT(request, response, log) {
                     routesUtils.responseNoBody(err, null, response, 200, log));
             } else if (request.query.acl === undefined) {
                 // PUT bucket
-                api.callApiMethod('bucketPut', request, log, (err) =>
+                if (request.post) {
```

Comment:
I don't know if it's the right place for this parsing, may be in a function where you create the requestContext can be better ? :)